src/services/network.service.ts

```diff
@@ -272,7 +277,35 @@ export class NetworkService implements OnApplicationBootstrap {
 
       const canUpdateEra = blockTime - eraStartTime.toNumber() > eraPeriod.toNumber();
       if (canUpdateEra) {
-        await this.sendTransaction('update era number', () => this.sdk.eraManager.safeUpdateAndGetEra());
+        await this.sendTransaction('update era number', async () =>
+          this.sdk.eraManager.safeUpdateAndGetEra(),
+        );
+      }
+    };
+  }
+
+  closeExpiredStateChannelsAction() {
+    const config = NETWORK_CONFIGS.kepler;
+    const client = new GraphqlQueryClient(config);
+
+    return async () => {
+      const apolloClient = client.explorerClient;
```

Comment:
[suggestion] Move query to a seperate function : `getExpiredStateChannels()` which return `unfinalisedPlans`